chapters/ch04.md

```diff
@@ -68,10 +79,11 @@ Many other details such as neural network architecture are also important, and t
 [1] This diagram shows NNs doing better in the regime of small datasets. This effect is less consistent than the effect of NNs doing well in the regime of huge datasets. In the small data regime, depending on how the features are hand-engineered, traditional algorithms may or may not do better. For example, if you have 20 training examples, it might not matter much whether you use logistic regression or a neural network; the hand-engineering of features will have a bigger effect than the choice of algorithm. But if you have 1 million examples, I would favor the neural network.
 
 ->
-
+[1] Mặc dù hình vẽ thể hiện mạng neural cho kết quả tốt hơn với tập dữ liệu nhỏ, nhưng nhận định này ít đồng thuận hơn so với nhận định về mạng neural với dữ liệu lớn. Vì đối với dữ liệu nhỏ, chất lượng thuật toán truyền thống có thể hoặc không cho kết quả tốt hơn mạng neural và bị phụ thuộc rất nhiều vào cách tạo đặc trưng thủ công. Nếu ta chỉ có 20 mẫu huấn luyện thì việc dùng hồi quy logistic hay mạng neural không khác biệt nhiều. Nhưng đồng thời, đặc trưng thủ công bị phụ thuộc nhiều vào thuật toán học máy. Còn nếu có một triệu mẫu, thì tôi sẽ chọn dùng mạng neural. 
```

Comment:
Từ 'consistent' dịch là 'nhất quán' thì đúng hơn; 'đồng thuận' gần với 'agree'. 'effect' cũng không phải là 'nhận định'. Hai câu đầu khá khó dịch, tớ đang nghĩ xem có cách nào hay hơn và chuẩn hơn không.

Chỗ " a neural network; the hand-engineering of features will have a bigger effect than the choice of algorithm." không nên tách ra làm hai câu vì nghĩ bị thay đổi. Tớ đề xuất dịch như lần trước tớ nói:

"không khác biệt nhiều; việc chọn đặc trưng thủ công sẽ mang lại nhiều sự khác biệt hơn".